packages/components/src/box-control/test/index.tsx

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { render, screen } from '@testing-library/react';
+import { fireEvent, render, screen } from '@testing-library/react';
```

Comment:
Added `fireEvent` due to `slider` actions not being supported by `UserEvent`: https://github.com/testing-library/user-event/issues/871

I did wonder if it was possible with a pointer event but I saw that for other sliders in the codebase, we are using `fireEvent`.